spec/models/news_spec.rb

```diff
@@ -54,5 +63,27 @@
     describe 'requires valid source' do
       it { should allow_value('http://foo.com').for(:source) }
     end
+
+    describe 'requires whitelist of values' do
+      it do
+        should validate_inclusion_of(:tech).in_array(%w(ruby rust go html css javascript))
+      end
+    end
+  end
+
+  context 'scopes' do
+    describe '.latest_by_tech' do
+      before do
+        (1..15).each do |i|
```

Comment:
WARNING: Lint/UnusedBlockArgument: Unused block argument - `i`. You can omit the argument if you don't care about it. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)